Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md

```diff
@@ -121,4 +121,8 @@ For the jumpbox you just created in the hub network to have access to Key vault'
 5. Select the hub virtual network for the **Virtual network** field
 6. Click on **OK** at the bottom
 
-:arrow_forward: [Deploy a Basic Workload](./08-workload.md)
+## Enable Blue Green Deployment
+
+If you want to activate the blue green deployment you can explore the [section](./11-blue-green.md)
```

Comment:
change to: "If you want to explore how to use a blue greed deployment, refer to this section [section](./11-blue-green.md)"

Reply:
I will remove, as new solution I can add the reference in the readme of the Blue Green dedicated folder.
![image](https://user-images.githubusercontent.com/48756634/187874883-0fe02343-342a-41d8-8b27-d4c6522f30b7.png)